utoipa-gen/src/component/schema.rs

```diff
@@ -1007,7 +1007,12 @@ impl ComplexEnum<'_> {
                     rename_all,
                 );
 
-                let example = pop_feature!(unit_features => Feature::Example(_));
+                let example: Option<Feature> = pop_feature!(unit_features => Feature::Example(_));
+
+                let description =
+                    CommentAttributes::from_attributes(&variant.attrs).as_formatted_string();
+                let description = (!description.is_empty())
+                    .then(|| Feature::Description(Description(description)));
```

Comment:
If desired I can extend the enum token stream with the correct tokenization instead of wrapping with `Feature::Description` (which needs the pub identifier `features.rs`). I like leveraging the existing source of truth for description token generation but can also see that maybe we don't want to use a `Feature` for a non-feature.

Reply:
Yeah less sources of truth there are the better. There are few other cases where a feature is being manually constructed from a value. I think this is fine but though better would be to implement `From` trait for the description feature. This way there is no need to make the value public.
```rust
impl From<String> for Description {
  fn from(value: String) -> Self {
    Self(value)
  }
}
```